docs/1.0. Certificate Provisioning.md

```diff
@@ -221,126 +225,128 @@ If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD
+(Bare reference, why?)
 
 #### Certificate Request Error Response
 
-If the EST Server returns any other HTTP response, the request has been unsuccessful, this could be caused by a malformed request, server side error or the EST Client not being authorised. The EST Client SHOULD restart the EST workflow with an alternative EST Server if present, else the EST Client SHOULD implement an exponential backoff algorithm, with a staring period of 1 second before retrying.
+If the EST Server returns any other HTTP response, the request has been unsuccessful. This could be caused by a malformed request, server-side error or the EST Client not being authorised. The EST Client SHOULD restart the EST workflow with an alternative EST Server if present, else the EST Client SHOULD implement an exponential backoff algorithm, with a starting period of 1 second before retrying.
 
-If the EST Client is performing a renewal and the request is unsuccessful, the EST Client SHOULD carry on using the existing TLS Certificate if still valid and re-attempt renewal after half of the remaining period of validity has elapsed. If the remaining validity is less than 1 day, the EST Client SHOULD implement an exponential backoff algorithm, with a staring period of 1 second before retrying.
+If the EST Client is performing a renewal and the request is unsuccessful, the EST Client SHOULD carry on using the existing TLS Certificate if still valid and reattempt renewal after half of the remaining period of validity has elapsed. If the remaining validity is less than 1 day, the EST Client SHOULD implement an exponential backoff algorithm, with a starting period of 1 second before retrying.
 
 ### Certificate Renewal
 
-Renewal of the TLS Certificate SHOULD be attempted no sooner than 50% of the certificate's expiry time or before the 'Not Before' date on the certificate. It is RECOMMENDED that certificate renewal is performed after 80% of the expiry time. If the EST Client's TLS Certificate is no longer valid then the [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow should be followed.
+Renewal of the TLS Certificate SHOULD be attempted no sooner than 50% of the certificate's expiry time or before the 'Not Before' (GOOD GOD!) date on the certificate. It is RECOMMENDED that certificate renewal is performed after 80% of the expiry time. If the EST Client's TLS Certificate is no longer valid, the [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow should be followed.
 
 The EST Client MUST generate a new key pair and CSR matching the TLS certificate it is being used to replace. The EST Client SHOULD make a request containing the CSR to the `/simplereenroll` endpoint of the EST Server. The EST Client MUST include the TLS Certificate being renewed during the TLS handshake with the EST Server.
 
 EST Client SHOULD handle the EST Server's response as per [Certificate Request Response](#certificate-request-response).
 
```

Comment:
Suggest that this passage from the "Notes for Implementers" section is either:

- better placed here
- better placed in the implementation guide
- better deleted
```suggestion

Care SHOULD be taken when implementing the certificate renewal process, to make sure the renewal does not affect user operation in a critical way (e.g. loss of control by connected NMOS Clients or the primary operation of the device).
```